phenol-core/src/main/java/org/monarchinitiative/phenol/ontology/data/ImmutableOntology.java

```diff
@@ -258,25 +264,28 @@ public static Builder builder() {
   }
 
   public static class Builder {
-    private Map<String, String> metaInfo = new LinkedHashMap<>();
-    private Collection<Term> terms = new ArrayList<>();
-    private Collection<Relationship> relationships = new ArrayList<>();
+    private final SortedMap<String, String> metaInfo = new TreeMap<>();
+    private final Collection<Term> terms = new ArrayList<>();
+    private final Collection<Relationship> relationships = new ArrayList<>();
+
+    private Builder() {
+    }
 
     public Builder metaInfo(Map<String, String> metaInfo) {
       Objects.requireNonNull(metaInfo);
-      this.metaInfo = new LinkedHashMap<>(metaInfo);
+      this.metaInfo.putAll(metaInfo);
```

Comment:
Don't do this - calling this twice now changes the behaviour and isn't in line with the method name. If you're going to allow adding create a new `metaInfoWith(Map<String, String> metaInfo)` or `metaInfoPut(String key, String value)`